integration/clientv3/naming/endpoints_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2016 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package naming_test
```

Comment:
For this test, it's disable because there is no entrypoint in our CI.

I think we have two ways to enable it

* https://github.com/etcd-io/etcd/blob/3663ae13fea8019dd8f2e1407412e52c9f9cb017/test#L194 Update `test` script to test `integration/clientv3/naming`
* Or move the test code into `clientv3/integration/` and update `test` script to enable the `clientv3/integration/naming`.

Both looks good to me.

Reply:
Code looks good to me and the first step is pretty safe because it only adds new function which no function uses.
Just want to ensure the new test can be covered by CI.

Thanks!

Reply:
Can it be as simple as below? I suggest to add it in this PR.

```
$ git diff
diff --git a/test b/test
index 260ea62c1..dfa6c2044 100755
--- a/test
+++ b/test
@@ -185,7 +185,7 @@ function integration_pass {
        # if TESTCASE not set, PKG set, run all test cases in specified package
        # if TESTCASE not set, PKG not set, run all tests in all integration and integration_extra packages
        if [ -z "${TESTCASE}" ] && [ -z "${USERPKG}" ]; then
-               go test -timeout "${USERTIMEOUT}" -v -cpu "${TEST_CPUS}" "$@" "${REPO_PATH}/integration"
+               go test -timeout "${USERTIMEOUT}" -v -cpu "${TEST_CPUS}" "$@" "${REPO_PATH}/integration/..."
                integration_extra "$@"
        else
                if [ -z "${USERPKG}" ]; then
```

Reply:
@fuweid and @ahrtr Thanks all!

I think it's better to move the test code into `clientv3/integration/` and enable the test with 

```
go test -timeout 25m -v ${RACE} -cpu "${TEST_CPUS}" "$@" "${REPO_PATH}/clientv3/integration/..."
```